mmstudio/src/main/java/org/micromanager/acquisition/internal/acqengjcompat/AcqEngJAdapter.java

```diff
@@ -840,9 +993,15 @@ public void setParentGUI(Studio parent) {
       studio_.events().registerForEvents(this);
    }
 
+   /**
+    * This is ignored by the Clojure engine, and also does not have any function
+    * in this engine.  Deprecate?  Do not rely on this to do anything.
+    *
+    * @param stageLabel Name of the focus drive to use.  Ignored.
+    */
    @Override
    public void setZStageDevice(String stageLabel) {
-      zstage_ = stageLabel;
+      zStage_ = stageLabel;
    }
```

Comment:
I would say deprecate. AcqEngJ right now just gets the Core Z device every time it controls hardware:

https://github.com/henrypinkard/AcqEngJ/blob/b676315feaae22611594f4de9f497e7030440f75/src/main/java/org/micromanager/acqj/internal/Engine.java#L482

Reply:
Agreed.  Let's coordinate with Mark, as this will need change to the Micro-Manager version (I believe).  We may take that opportunity to evaluate more of this api.